mlrun/feature_store/api.py

```diff
@@ -237,14 +238,14 @@ def get_online_feature_service(
 
             Example with imputing::
 
-                with get_online_feature_service(vector_uri, impute_policy={"*": "$mean", "amount": 0)) as svc:
+                with get_online_feature_service(vector_uri, ['id'], impute_policy={"*": "$mean", "amount": 0)) as svc:
```

Comment:
```suggestion
                with get_online_feature_service(vector_uri, entity_keys=['id'], impute_policy={"*": "$mean", "amount": 0)) as svc:
```

Since you moved the parameter to the end, you need to adjust the examples.